src/board-explorer-app/board-explorer-app.html

```diff
@@ -929,7 +935,7 @@
       }
 
      Array.prototype.forEach.call(this.$.layout.querySelectorAll("paper-button[match='" + item.refdes + "']"), function(el) {
-        el.disabled = false;
+        el.setAttribute('selected', '');
```

Comment:
@juliocri Since you're using the attribute assignment on the instantiation of paper-button in the DOM, you should be able to set the property on the element via el.selected = "" instead of using setAttribute. I was using setAttribute in my sample because I wasn't setting an item's property.

Reply:
I sent a couple of changes, I tried to  use ```el.selected = ""```  but does not work for me, I think is because is a DOM, ```setAttribute``` is the only method that already set the property in the paper button.